src/pages/Aboutpage.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import AOS from 'aos';
+import 'aos/dist/aos.css';
+import TeamMember from '../components/TeamMember';
+import './Aboutpage/Aboutpage.css';
+const Aboutpage = () => {
+  useEffect(() => {
+    AOS.init({
+      duration: 2000,
+    });
+  });
+  return (
+    <>
+      <div
+        className="aboutpage  w-full py-24 px-64 text-center text-base
+            overflow-hidden bg-gray-700 h-64 "
+        data-aos="zoom-out"
+      >
+        Lorem Ipsum is simply dummy text of the printing and typesetting
```

Comment:
lets add some real text here and make sure its centered cause it is not
![image](https://user-images.githubusercontent.com/12795240/89955023-cdf7bb80-dc3a-11ea-9794-22fca8005dd3.png)